src/commands/apphosting-config-export.ts

```diff
@@ -26,28 +30,36 @@ export const command = new Command("apphosting:config:export")
     const currentDir = process.cwd();
 
     // Get all apphosting yaml files ignoring the apphosting.local.yaml file
-    const yamlFilePaths = allYamlPaths(currentDir)?.filter(
-      (path) => !path.endsWith(APPHOSTING_LOCAL_YAML),
+    const yamlFilePaths = discoverConfigsInProject(currentDir)?.filter(
+      (path) => !path.endsWith(APPHOSTING_LOCAL_YAML_FILE),
     );
 
     if (!yamlFilePaths) {
-      logger.warn("No apphosting YAMLS found");
+      logger.warn("No apphosting.yaml found");
       return;
     }
 
     // Load apphosting.local.yaml file if it exists. Secrets should be added to the env list in this object and written back to the apphosting.local.yaml
-    const apphostingLocalConfigPath = yamlPath(currentDir, APPHOSTING_LOCAL_YAML);
-    const localAppHostingConfig = await loadAppHostingYaml(apphostingLocalConfigPath ?? undefined);
+    const localApphostingConfigPath = yamlPath(currentDir, APPHOSTING_LOCAL_YAML_FILE);
+    let localAppHostingConfig: AppHostingYamlConfig = AppHostingYamlConfig.empty();
 
-    const configsToExport = await getConfigToExport(yamlFilePaths);
-    const secretsToExport = configsToExport.secrets;
+    if (localApphostingConfigPath) {
+      localAppHostingConfig = await AppHostingYamlConfig.loadFromFile(localApphostingConfigPath);
+    }
```

Comment:
nit: try to declare variables close to where they're used. 

go/java-style#s4.8.2.2-variables-limited-scope

For example:

```
var a;
<15 lines of code>
a += 1
```

```
<15 lines of code>
var a;
a += 1;
```

In the second example, it's much easier to see there's a bug